at-a-glance.rst

```diff
@@ -1,43 +1,41 @@
 Mink at a Glance
 ================
 
-There's huge number of browser emulators out there, like `Goutte`_, `Selenium`_,
-`Sahi`_ and others. They all do the same job, but do it very differently.
-They behave differently and have very different API's. But, what's more important,
-there is actually 2 completely different types of browser emulators out there:
+There are a huge number of browser emulators out there, like `Goutte`_, `Selenium`_,
+`Sahi`_ and others. They all do the same job but they behave differently and have very different API's. What's more important is that there are actually two completely different types of browser emulators out there:
```

Comment:
Please keep the line wrapped after the first word crossing 72 chars (we follow the same standard that the symfony-docs project for that)